package.json

```diff
@@ -63,7 +63,12 @@
     "@azure/msal-node": "^1.18.4",
     "axios": "^1.7.2",
     "tar": "6.1.9",
-    "glob-parent": "5.1.2"
+    "glob-parent": "5.1.2",
+    "lodash": "4.17.21",
+    "lodash.pick": "https://github.com/lodash/lodash/archive/refs/tags/4.17.21.tar.gz"
```

Comment:
We could use `npm:` like in this pr https://github.com/southworks/botbuilder-js/pull/433/files.